app/src/main/java/de/tum/in/tumcampusapp/component/ui/transportation/widget/MVVWidget.java

```diff
@@ -140,6 +140,8 @@ static void updateAppWidget(Context context, AppWidgetManager appWidgetManager,
         RemoteViews rv = new RemoteViews(context.getPackageName(), R.layout.mvv_widget);
         rv.setTextViewText(R.id.mvv_widget_station, widgetDepartures.getStation());
 
+        // TODO Offline layout
```

Comment:
Is this still a TODO you want to do with this PR?

Reply:
Forgot to remove that comment. Fixed now.